### PR TITLE
Quantiles

### PR DESCRIPTION
**Pull request by AlexanderSaydakov**

### CODE REVIEW

**Comment by AlexanderSaydakov** [comment]
I think it is ready, but should wait for the datasketches-cpp release

File: src/quantiles_double_sketch_pg_functions.c [hunk 1]
```diff
@@ -0,0 +1,384 @@
...
+
+  split_points = palloc(sizeof(double) * arr_len_in);
+  for (i = 0; i < arr_len_in; i++) {
+    split_points[i] = DatumGetFloat8(data_in[i]);
+  }
+  result = (Datum*) quantiles_double_sketch_get_pmf_or_cdf(sketchptr, split_points, arr_len_in, false, false);
+  pfree(split_points);
+
+  // construct output array of fractions
+  arr_len_out = arr_len_in + 1; // N split points devide the number line into N+1 intervals
```
Comment (jmalkin) [inline src/quantiles_double_sketch_pg_functions.c]: typo: divide

File: src/quantiles_double_sketch_pg_functions.c [hunk 2]
```diff
@@ -0,0 +1,384 @@
...
+
+  split_points = palloc(sizeof(double) * arr_len_in);
+  for (i = 0; i < arr_len_in; i++) {
+    split_points[i] = DatumGetFloat8(data_in[i]);
+  }
+  result = (Datum*) quantiles_double_sketch_get_pmf_or_cdf(sketchptr, split_points, arr_len_in, true, false);
+  pfree(split_points);
+
+  // construct output array of fractions
+  arr_len_out = arr_len_in + 1; // N split points devide the number line into N+1 intervals
```
Comment (jmalkin) [inline src/quantiles_double_sketch_pg_functions.c]: same typo

**Review by jmalkin — APPROVED** [review]
👍

**Pull request merged**